definitions/fenix.toml

```diff
@@ -471,3 +500,161 @@ submission_date_column = "submission_date"
 [data_sources.new_profile_activation]
 from_expression = "`moz-fx-data-shared-prod.fenix.new_profile_activation`"
 experiments_column_type = "none"
+
+[data_sources.turn_on_notifications]
+from_expression = """(
+SELECT
+    expo.submission_date
+    , expo.client_id
+    , case when conv.client_id is not null then 1 else 0 end as converted_flag
+FROM (
+      SELECT
+            client_info.client_id as client_id
+            , min(DATE(submission_timestamp)) as submission_date
+      FROM
+        mozdata.org_mozilla_firefox.events tm
+      CROSS JOIN
+        UNNEST(events) AS event
+      CROSS JOIN
+        UNNEST(ping_info.experiments) AS expr
+      WHERE event.category = "onboarding" AND expr.key = '{experiment_slug}'
```

Comment:
metric-hub metrics also have use outside of experiments, so I wouldn't include this here

Reply:
Sure I will remove that from the join